### PR TITLE
Feature/crop mask product config

### PR DESCRIPTION
**Pull request by fivejjs**
add crop-mask product yaml. Locally validated in postgres docker

### CODE REVIEW

File: products/crop_mask_product.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    units: '1'
+
+  - name: PROB
+    aliases: ['crop_prob']
+    dtype: uint8
+    nodata: 255
+    units: '1'
+
+storage:
+  crs: EPSG:6933
```
Comment (Kirill888) [inline products/crop_mask_product.yaml:26]: Let's use lower-case for epsg to be compatible with ARD data in AU.

Comment (fivejjs) [reply products/crop_mask_product.yaml]: sure.

File: products/crop_mask_product.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+name: crop_mask
+description: prediction based s2 l2a gm_tmad, NDVI, LAI, MNDWI, rainfall, slope
```
Comment (alexgleith) [inline products/crop_mask_product.yaml:3]: Please use a nice sentence to describe the product. This doesn't make sense.

File: products/crop_mask_product.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+metadata_type: eo3
+
+license: CC-BY-4.0
+
+metadata:
+  product:
+    name: crop_mask
+
+measurements:
+  - name: MASK
```
Comment (alexgleith) [inline products/crop_mask_product.yaml:13]: I would prefer lower case here

File: products/crop_mask_product.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    nodata: 255
+    units: '1'
+
+  - name: PROB
+    aliases: ['crop_prob']
+    dtype: uint8
+    nodata: 255
+    units: '1'
+
+storage:
```
Comment (alexgleith) [inline products/crop_mask_product.yaml:25]: Storage above the measurements please.

**Review by alexgleith — CHANGES_REQUESTED** [review]
Why are there 31 commits?

This is a single, simple file!

File: products/crop_mask_product.yaml (hunk 3 above)
Comment (fivejjs) [reply products/crop_mask_product.yaml:13]: I am refering s2_l2a band "B01" like name.

```
measurements:
  - name: "B01"
    aliases: [band_01, coastal_aerosol]
    units: "1"
    dtype: uint16
    nodata: 0
```

Comment (fivejjs) [reply products/crop_mask_product.yaml:13]: I already generating the data with such naming pattern.

**Comment by fivejjs** [comment]
messed up the commits. close this branch and create a new one.

**Pull request closed without merging**